<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MANUAL SWITCHING</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>0</y>
      <width>31</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PROGRAM SWITCHING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_s1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>93</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_s2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>93</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>II</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_s3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>93</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>III</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_s4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>93</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r7">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_r8">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_prog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="b_restart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pause">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>620</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PAUSED</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_port">
    <property name="title">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_port"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
